test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -842,6 +842,18 @@ add_specs suite_builder =
             Math.min (Decimal.new "12E70") (Decimal.new "13E70") . should_equal (Decimal.new "12E70")
             Math.max (Decimal.new "12E70") (Decimal.new "13E70") . should_equal (Decimal.new "13E70")
 
+    suite_builder.group "Integer/Float .to_decimal" group_builder->
+        group_builder.specify ".to_decimal should convert to Decimal" <|
+            12 . to_decimal . should_be_a Decimal
+            12.3 . to_decimal . should_be_a Decimal
+
+        group_builder.specify "Float.to_decimal should attach a warning" <|
+            Problems.expect_warning Loss_Of_Numeric_Precision (12.3 . to_decimal)
```

Comment:
I'm wondering if the explicit conversion should still have the warning? I guess if you do it explicitly maybe it's not needed?

Reply:
@jdunkerley has said that because people are often unfamiliar with the fact that `Float` is approximate (for example, not being able to represent 0.1 exactly), that we need these warnings always. But I think it's something we can discuss.

Reply:
OK, that's fair enough